WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -243,10 +243,8 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
 + (id<FBResponsePayload>)handlePressButtonCommand:(FBRouteRequest *)request
 {
   NSError *error;
-  NSNumber *requestedDuration = request.arguments[@"duration"];
-  NSTimeInterval duration = (requestedDuration ? requestedDuration.doubleValue : -1);
   if (![XCUIDevice.sharedDevice fb_pressButton:(id)request.arguments[@"name"]
-                                   forDuration:duration
+                                   forDuration:(NSNumber *)request.arguments[@"duration"]
```

Comment:
would this typecast work if the argument is nil?

Reply:
yes, then here was `nil`